client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -355,10 +360,14 @@ class PurchasesListing extends Component {
 	}
 }
 
+const getSiteLevelManagePurchaseUrlFor = ( targetSiteSlug, targetPurchaseId ) =>
```

Comment:
I think we could import `getManagePurchaseUrlFor` from `calypso/my-sites/purchases/paths` and use that instead of duplicating it here.